notebooks/codesearchnet-opennmt.py

```diff
@@ -1,36 +1,39 @@
+"""
+CLI tool for converting CodeSearchNet dataset to OpenNMT format for
+function name suggestion task.
+
+Usage example:
+    wget 'https://s3.amazonaws.com/code-search-net/CodeSearchNet/v2/java.zip'
+    unzip java.zip
+    python notebooks/codesearchnet-opennmt.py \
+        --data_dir='java/final/jsonl/valid' \
+        --newline='\\n'
```

Comment:
```suggestion
```
`\\n` is already the default

Reply:
If that's ok, I'd like to keep it as illustrative example of the transformation this script is doing